### PR TITLE
Bring thank you text in the center

### LINKED ISSUE #150: Bring thank you text in the center
On /submit_contest/ page, bring the text in the center of the page i.e., both horizontally and vertically in the center of the page.
thank you page

Also replace *Y* with *y*.

### PR DESCRIPTION
**Pull request by kanishkarj**
**Checklist**

- [ ] My branch is up-to-date with the upstream `develop` branch.
- [ ] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: fixes #150
This PR brings the thank you text to the center and replaces Y with y.

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@kanishkarj Thanks for contributing!! 😄 As i am seeing that you are vary enthusiast to contribute in this project but as [d3vil0p3r](https://github.com/d3vil0p3r) already fixed the issue so it's fair to give hime the chance and merge his PR 🚀 .  

Hey!! This is not the end of open source development, you can look into existing [issues](https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/issues) and try to fix the issue. In case you stuck anywhere just message on the slack channel (Request you invite [here](https://opensourcehelp.herokuapp.com)). Anyone who are online will help you with the issues you are facing, alternatively you can search on google and stackoverflow. Maybe people had faced the same issue you faced and provided the solution.

Happy Coding!!

File: oshc/main/templates/contest_submission.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 <div class="jumbotron text-xs-center">
   <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
+      <h1 class="display-3 text-center">Thank you!</h1>
```
Comment (tapaswenipathak) [inline oshc/main/templates/contest_submission.html:7]: This isn't working for me on my local or trial deployment. I have cache disabled. Can you please share screenshots?

**Review by tapaswenipathak — COMMENTED** [review]
.

**Comment by tapaswenipathak** [comment]
Someone fixed this issue with 8c1d068c75437e3a2f5e46335d49c0b82278d7a6, so closing this one. Feel free to pick any other open issues or raise new.

**Pull request closed without merging**